requirements.in

```diff
@@ -11,4 +11,5 @@ paramiko==2.10.3
 pyparsing==2.4.7
 requests==2.27.0
 twine==3.2.0
-vcsinfo>=2.1.105
\ No newline at end of file
+vcsinfo>=2.1.105
+graphlib-backport==1.0.3
```

Comment:
This could probably be left open-ended at `graphlib-backport>=1.0.3`. The way I've been thinking about this (but haven't communicated it) is that the requirements.in contains known dependencies, with handling for specific versions where necessary, otherwise they are left a little more open-ended. Requirements.txt obviously contains the locked versions. It's not necessarily done this way, however, so this is fine to leave. Just communicating some things I've been pondering recently.

Reply:
I'll can change it, I was debating whether to leave it open or not.